src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/services/ingest.js

```diff
@@ -0,0 +1,58 @@
+import uiModules from 'ui/modules';
```

Comment:
This file can be merged with the existing service in `ui/public/ingest`. Also when creating modules with Angular services it's preferable to just return the factory function so that the module can be used with the `Private` module loader.